tests/test_metaculus.py

```diff
@@ -212,3 +215,32 @@ def test_get_community_prediction_log(metaculus_questions):
 def test_sample_community_binary(metaculus_questions):
     value = metaculus_questions.binary_question.sample_community()
     assert bool(value) in (True, False)
+
+
+def test_show_prediction_with_fitted(
```

Comment:
I'm not a big fan of just testing that the code runs without erroring (as opposed to actually testing that it does the desired thing). I think we should just get rid of these tests

But if you feel like we should keep them, add docstrings explaining why

Reply:
(TBC I am grateful that you included these for the sake of this PR, so we can see that #408 is indeed fixed, but I don't think it's worth keeping them)